editing/edit-with-branch-versioning/src/main/java/com/esri/samples/edit_with_branch_versioning/EditWithBranchVersioningController.java

```diff
@@ -92,10 +92,10 @@ public void initialize() {
 
       // create a service geodatabase
       serviceGeodatabase = new ServiceGeodatabase("https://sampleserver7.arcgisonline" +
-        ".com/arcgis/rest/services/DamageAssessment/FeatureServer");
+        ".com/server/rest/services/DamageAssessment/FeatureServer");
 
       // set the user credentials required to authenticate with the service geodatabase
-      UserCredential userCredential = new UserCredential("editor01", "editor01.password");
+      UserCredential userCredential = new UserCredential("editor01", "S7#i2LWmYH75");
```

Comment:
This looks like quite a 'serious' password (as opposed to the previous one), are we sure it's supposed to be public? I'm confident this didn't get put out by mistake, but best to make triple sure?

Reply:
Thanks for checking! This is the same for all of the platform PRs, these were the requested changes.

Reply:
Yep, this caught me for a second too. Yes all these samples should have hardcoded credentials as per the pattern shown here 👍